CMakeLists.txt

```diff
@@ -71,43 +71,43 @@ install(FILES ${QM_FILES} DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/${ORGANIZATIO
 install(FILES data/${DESKTOP_FILE} DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/applications)
 
 ecm_install_icons(ICONS
-    data/icons/freedesktop/128-apps-intel.svg
+    data/icons/freedesktop/128-apps-integrated.svg
```

Comment:
I think we should just use another icon for AMD.

Reply:
> I think we should just use another icon for AMD.

Yeah I agree

Reply:
It just should be taken from [this](https://github.com/masalla-art/masalla-icon-theme) theme (as Intel and Nvidia icons).